invenio_records_resources/services/records/service.py

```diff
@@ -170,16 +170,27 @@ def create(self, identity, data, links_config=None):
         return self._create(
             self.record_cls, identity, data, links_config=links_config)
 
-    def _create(self, record_cls, identity, data, links_config=None):
+    def _create(self, record_cls, identity, data, links_config=None,
+                raise_errors=True):
         """Create a record.
 
         :param identity: Identity of user creating the record.
         :param data: Input data according to the data schema.
+        :param links_config: Links configuration.
+        :param partial: Marshmallow 'partial' argument.
```

Comment:
minor: partial is not used anywhere (nor accepted by the function definition).

Reply:
Thanks! That's a leftover.